projects/demo/src/modules/components/input-date/index.html

```diff
@@ -9,20 +9,24 @@
             – input with a calendar.
         </div>
 
-        <h3>DI-tokens for date localization:</h3>
+        <h3>
```

Comment:
Looks like this piece repeats a lot, should we make a component for it?

Reply:
made a link to the token page like NumberFormat